todo_app/app.py

```diff
@@ -1,11 +1,15 @@
-from flask import Flask
-
+from flask import Flask, render_template, request, redirect
 from todo_app.flask_config import Config
+from todo_app.data.session_items import *
```

Comment:
minor, but its normally better to explicitly import rather than importing `*`

From the python docs: https://docs.python.org/3/tutorial/modules.html#importing-from-a-package

> Although certain modules are designed to export only names that follow certain patterns when you use import *, it is still considered bad practice in production code.